charts/ocis/values.yaml

```diff
@@ -1238,8 +1251,41 @@ services:
   web:
     # Configuration for web, that ends up in the config.json file.
     config:
-      # -- Disables the feedback link in the Web UI.
-      disableFeedbackLink: false
+      contextHelpersReadMore:
+        # -- Specifies whether the “Read more” link should be displayed or not.
+        enabled: true
+      feedbackLink:
+        # -- Enables the feedback link in the Web UI.
+        enabled: true
```

Comment:
Does it make sense to have the `feedbackLink` enabled by default with not `href` set?

Reply:
Ok